RELEASE.md

```diff
@@ -0,0 +1,29 @@
+# How to create new release
+
+## Manual version using Packit
+
+```
+$ ./bump-version.sh
+# Check that the spec file is correctly modified
+# Create new commit from this change; this commit will become the new tag
+
+# Create new release on Github containing the number of this release as a
+# name, the same number as a tag, and description copied from the previous
+# one.
+
+$ packit status
+# This should show a new release available in upstream
+# In order to execute this you will need a ~/.config/packit.yaml containing
+# your FAS account, Pagure token, Github token, and valid Kerberos TGT for
+# FEDORAPROJECT.ORG realm
+$ packit propose-update
+# It will create a PR on src.fedoraproject.org
+# If everything looks fine, you can merge it
```

Comment:
Oh I can merge myself? Awesome :)

Reply:
If we want to do reviews we might need more maintainers. WDYT @ondrejbudai and @jkozol? :)